### PR TITLE
[MC1, MC2]: Set up drag and drop modal

### PR DESCRIPTION
**Pull request by liviaw**
In this PR, the changes are adding css and stylings into the home page and modal
Screen Shot 2021-01-14 at 3 36 15 pm

### CODE REVIEW

File: src/Components/ImportModal/ImportModal.module.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  .dropModal {
+    background-color: rgb(0, 196, 204, 50%);
+    width: 90vw;
+    min-height: 90vh;
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    position: relative;
+    margin: 10px 0;
+
```
Comment (juliamitchelmore) [inline src/Components/ImportModal/ImportModal.module.css]: remove

File: src/Components/ImportModal/ImportModal.module.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    margin: 0 auto;
+    background-size: cover;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+  }
+  
+  .dropModal {
+    background-color: rgb(0, 196, 204, 50%);
```
Comment (juliamitchelmore) [inline src/Components/ImportModal/ImportModal.module.css]: Would be good to set up variables for these colours

**Review by juliamitchelmore — APPROVED** [review]
👍 
The title of this PR could be updated to `Style drag and drop modal`, as that's what the PR is handling

File: src/Components/ImportModal/ImportModal.module.css (hunk 2 above)
Comment (liviaw) [reply src/Components/ImportModal/ImportModal.module.css]: yepp will do, only used the canva brand colors. I will make this change at a later PR

Comment (liviaw) [reply src/Components/ImportModal/ImportModal.module.css]: yepp will do, only used the canva brand colors. I will make this change at a later PR

File: src/Components/ImportModal/ImportModal.module.css (hunk 1 above)
Comment (liviaw) [reply src/Components/ImportModal/ImportModal.module.css]: ah the extra line?

**Pull request merged**